packages/garbo/src/diet.ts

```diff
@@ -150,17 +187,36 @@ function consumeSafe(
     throw "No spleen to clear with this.";
   }
   const averageAdventures = getAverageAdventures(item);
-  if (!skipAcquire && (averageAdventures > 0 || additionalValue)) {
+  const usingMoonZoneRestaurant = availableFromMoonZoneRestaurant(item);
+  if (
+    !skipAcquire &&
+    !usingMoonZoneRestaurant &&
+    (averageAdventures > 0 || additionalValue)
+  ) {
     const cap = Math.max(0, averageAdventures * MPA) + (additionalValue ?? 0);
     acquire(qty, item, cap, true);
-  } else if (!skipAcquire) {
+  } else if (!skipAcquire && !usingMoonZoneRestaurant) {
```

Comment:
If this condition is true for both consecutive ifs, you should either:
- resequence the ifs to move your else up top, if that makes sense
- make a big if with the shared statement and a smaller if...else within